sdk/resourcemanager/Azure.ResourceManager.Core/samples/README.md

```diff
@@ -0,0 +1,16 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-resource-manager
+name: Azure.ResourceManager.Core samples for .NET
+description: Samples for the Azure.ResourceManager.Core client library
+---
+
+# Azure.ResourceManager.Core Samples
+
+- [Getting a subscription](Sample1_HelloWorld.md)
```

Comment:
Lets have the text mention this "Hello World" as well to indicate this isn't a real world scenario example but a hello world example